devito/data/allocators.py

```diff
@@ -306,6 +307,55 @@ def put_local(self):
         return self._node == 'local'
 
 
+class ExternalAllocator(MemoryAllocator):
+
+    """
+    External allocator for definig Function symbols with data loaded by the
+    user, without copying it to Devito's allocated memory.
+
+    Parameters
+    ----------
+    numpy_array : numpy.array
```

Comment:
array-like  instead of numpy.array ?

array instead of numpy_array ?

and then in the description, "Any object exposing the buffer interface, such as a numpy.ndarray"

Reply:
thank you, fixed